server.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable */
+const express = require('express');
+const webpack = require('webpack');
+const webpackDevMiddleware = require('webpack-dev-middleware');
+
+const app = express();
+const config = require('./webpack.config.js');
+const compiler = webpack(config);
```

Comment:
you need to change const ... = require(...) to import ... from ... for better Javascript practices